src/model/topic.ts

```diff
@@ -46,22 +104,15 @@ export type Topic = Owned & {
   id: string
   page: string
   private: boolean
-  // FIXME: remove / fix after API is migrated to elements
-  reuses?: Rel | null
-  datasets?: Rel | null
-  elements?: Rel | null
+  elements: Rel
   slug: string
   tags: string[]
   uri: string
   spatial: SpatialField | undefined
 }
 
-export type TopicPostData = Omit<
-  Topic,
-  'datasets' | 'reuses' | 'id' | 'slug'
-> & {
+export type TopicPostData = Omit<Topic, 'elements' | 'id' | 'slug'> & {
   id?: string
   slug?: string
-  datasets: string[]
-  reuses: string[]
+  elements?: Factor[]
```

Comment:
```suggestion
  elements?: GenericElement[]
```
?

Reply:
I still find it slightly confusing since it's in a generic model, but ok, better keep the stricter bound for now, I suppose.